startup/gui/lightFilterVis.py

```diff
@@ -0,0 +1,2 @@
+import ctypes
+ctypes.cdll.LoadLibrary( 'libGafferArnoldUI.so' )
```

Comment:
This doesn't work on Mac, where the extension is ".dylib". I think all our modules contain some bindings at this point in time, but in the past when we've had the same issue, I've just made an empty module that is linked to the necessary library. `import` is the canonical way of getting functionality into Python, so I think it makes sense to continue to use it for this purpose.